File: ios/Classes/Models/HMSResultExtension.swift
//
//  HMSResultExtension.swift
//  hmssdk_flutter
//
//  Created by Pushpam on 27/03/23.
//

import Foundation

class HMSResultExtension {

    static func toDictionary(_ success: Bool, _ data: Any?) -> [String: Any?] {
        let dict = [
            "success": success,
            "data": data
        ]
        return dict
    }

}

File: packages/hms_video_plugin/android/src/main/kotlin/live/hms/hms_video_plugin/extension/HMSResultExtension.kt
package live.hms.hms_video_plugin

class HMSResultExtension {
    companion object {
        fun toDictionary(
            success: Boolean,
            data: Any? = null,
        ): HashMap<String, Any?> {
            val hashMap = HashMap<String, Any?>()
            hashMap["success"] = success
            hashMap["data"] = data
            return hashMap
        }
    }
}